build.sbt

```diff
@@ -1,13 +1,24 @@
 import sbt._
 
+// https://github.com/sbt/sbt-git
+enablePlugins(GitVersioning)
+
 lazy val commonSettings = Seq(
+  // sbt-git provides the isSnapshot.value variable, which will return true if there
+  // are no tags associated with the HEAD commit, or if there are uncommitted
+  // changes. However, sbt-git only appends the -SNAPSHOT suffix if there are
+  // uncommitted changes in the workspace. 
+  //
+  // https://github.com/sbt/sbt-git/blob/f8caf9365be380cf101e9605af159b5e7f842d0c/src/main/scala/com/typesafe/sbt/SbtGit.scala#L173
+  version := {
+    // Avoid Cyclic reference involving error

```

Comment:
This isn't super pretty, but when I was referencing `isSnapshot.value` directly, I was getting a `Cyclic reference involving` error.